protocol-definitions/Map.yaml

```diff
@@ -2942,3 +2942,27 @@ methods:
           since: 2.0
           doc: |
             old value of the entry
+  - id: 72
+    name: replaceAll
+    since: 2.0
```

Comment:
We should make the version since 2.4. Make sure that you put the version correctly on the `params` as well.